stabilizer/src/main/java/com/hazelcast/stabilizer/test/utils/PropertyBindingSupport.java

```diff
@@ -51,50 +39,75 @@ public static void bindProperties(Object test, TestCase testCase) throws NoSuchF
                 continue;
             }
 
-            //we also ignore probe- properties
+            // we also ignore probe-properties
             if (property.startsWith("probe-")) {
                 continue;
             }
 
-            bindProperty(test, property, value);
+            bindProperty(testInstance, property, value, optionalProperties);
         }
     }
 
-    public static void bindProperty(Object object, String property, String value) throws IllegalAccessException {
+    public static void bindOptionalProperty(Object instance, TestCase testCase, String propertyName)
```

Comment:
Can you add some javadoc? I know that there is not much, but I'm a bit curious about the advantage of this method.